examples/python/rgbd/rgbd.py

```diff
@@ -175,22 +175,30 @@ def main() -> None:
         args,
         "rerun_example_rgbd",
         default_blueprint=rrb.Horizontal(
-            rrb.Spatial3DView(name="3D", origin="world"),
+            rrb.Vertical(
+                rrb.Spatial3DView(name="3D", origin="world"),
+                rrb.TextDocumentView(name="Description", origin="/description"),
+                row_shares=[0.7, 0.3],
```

Comment:
```suggestion
                row_shares=[3, 1],
```
All the other row & column shares in here are whole numbers. Before I saw that I didn't care, but here's a pattern established that we should stick to.
(Numbers I put in is not the exact same, so please test first if this looks ok)

Reply:
`[7, 3]` should be closer to the original :P  (or `[2, 1]`)